shell/src/main/java/tachyon/shell/command/AbstractAclCommand.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.shell.command;
+
+import java.io.IOException;
+
+import tachyon.TachyonURI;
+import tachyon.client.file.TachyonFileSystem;
+import tachyon.client.file.options.SetAclOptions;
+import tachyon.conf.TachyonConf;
+import tachyon.exception.TachyonException;
+
+/**
+ * Parent class for commands: chown, chownr, chgrp, chgrpr, chmod and chmodr.
+ */
+public abstract class AbstractAclCommand extends AbstractTfsShellCommand {
```

Comment:
if we take `-R` option for chown, chgrp, chrmod, then we probably don't need this base class any more. :)